### ISSUE TITLE
posix_net_conf.sh set with invalid irq number

### ISSUE THREAD

**Issue by amoskong** [scylladb/scylla#2731]
*Installation details*
Scylla version (or git commit hash): 1.7.4-0.20170726.ff643e3
Cluster size: 1
OS (RHEL/CentOS/Ubuntu/AWS AMI): Ubuntu 14.04

*Description*
Found an error in starting stage, it don't block anything:
`/usr/lib/scylla/posix_net_conf.sh: line 33: /proc/irq//var/lib/scylla/smp_affinity: No such file or directory`

GCE instance type: n1-highmem-2 (2 vCPUs, 13 GB memory)

```
scylla-test@amos-ubuntu14:~$ lspci |grep Eth
00:04.0 Ethernet controller: Red Hat, Inc Virtio network device
```

```
Scylla version 1.7.4-0.20170726.ff643e3 starting ...
Setting a physical interface eth0...
Setting mask 00000001 in /proc/irq/24/smp_affinity
Setting mask 00000001 in /proc/irq/25/smp_affinity
Setting mask 00000001 in /proc/irq/26/smp_affinity
Setting mask 00000002 in /proc/irq/27/smp_affinity
Setting mask 00000002 in /proc/irq/28/smp_affinity
Setting mask 00000002 in /proc/irq//var/lib/scylla/smp_affinity
/usr/lib/scylla/posix_net_conf.sh: line 33: /proc/irq//var/lib/scylla/smp_affinity: No such file or directory
Setting mask 00000001 in /sys/class/net/eth0/queues/tx-0/xps_cpus
Setting mask 00000002 in /sys/class/net/eth0/queues/tx-1/xps_cpus
hugeadm:WARNING: Directory /var/lib/hugetlbfs/pagesize-2MB is already mounted.
hugeadm:WARNING: Directory /var/lib/hugetlbfs/pagesize-1GB is already mounted.
tuning /sys/devices/virtual/block/md0
already tuned: /sys/devices/virtual/block/md0/queue/scheduler
already tuned: /sys/devices/virtual/block/md0/queue/nomerges
tuning /sys/devices/pci0000:00/0000:00:03.0/virtio0/host0/target0:0:2/0:0:2:0/block/sdb
already tuned: /sys/devices/pci0000:00/0000:00:03.0/virtio0/host0/target0:0:2/0:0:2:0/block/sdb/queue/scheduler
already tuned: /sys/devices/pci0000:00/0000:00:03.0/virtio0/host0/target0:0:2/0:0:2:0/block/sdb/queue/nomerges
tuning /sys/devices/virtual/block/md0
```

```
           CPU0       CPU1   
  0:         44          0   IO-APIC   0-edge      timer
  1:         10          0   IO-APIC   1-edge      i8042
  4:       4550          0   IO-APIC   4-edge      serial
  8:          1          0   IO-APIC   8-edge      rtc0
  9:          0          0   IO-APIC   9-fasteoi   acpi
 12:        155          0   IO-APIC  12-edge      i8042
 24:          0          0   PCI-MSI 65536-edge      virtio1-config
 25:       4493          0   PCI-MSI 65537-edge      virtio1-input.0
 26:       5218          0   PCI-MSI 65538-edge      virtio1-output.0
 27:          1      11903   PCI-MSI 65539-edge      virtio1-input.1
 28:          1       8389   PCI-MSI 65540-edge      virtio1-output.1
 29:          0          0   PCI-MSI 49152-edge      virtio0-config
 30:          0          0   PCI-MSI 49153-edge      virtio0-control
 31:          0          0   PCI-MSI 49154-edge      virtio0-event
 32:     479640          0   PCI-MSI 49155-edge      virtio0-request
NMI:          0          0   Non-maskable interrupts
LOC:     613705     641723   Local timer interrupts
SPU:          0          0   Spurious interrupts
PMI:          0          0   Performance monitoring interrupts
IWI:          0          0   IRQ work interrupts
RTR:          0          0   APIC ICR read retries
RES:     741928     647620   Rescheduling interrupts
CAL:        552        489   Function call interrupts
TLB:     111637     205034   TLB shootdowns
TRM:          0          0   Thermal event interrupts
THR:          0          0   Threshold APIC interrupts
DFR:          0          0   Deferred Error APIC interrupts
MCE:          0          0   Machine check exceptions
MCP:          6          6   Machine check polls
ERR:          0
MIS:          0
PIN:          0          0   Posted-interrupt notification event
PIW:          0          0   Posted-interrupt wakeup event
```

@syuu1228

**Comment by slivne** [comment]
@vladzcloudius ?

**Comment by slivne** [comment]
@vladzcloudius  - ping

**Comment by vladzcloudius** [comment]
@slivne To begin with, 2.0 seem to work just fine with GCE:
```
 sudo /usr/lib/scylla/posix_net_conf.sh eth0
irqbalance is not running
Setting a physical interface eth0...
Setting mask 00000001 in /proc/irq/26/smp_affinity
Setting mask 00000001 in /proc/irq/24/smp_affinity
Setting mask 00000001 in /proc/irq/28/smp_affinity
Setting mask 00000002 in /proc/irq/25/smp_affinity
Setting mask 00000002 in /proc/irq/27/smp_affinity
Setting mask 00000003 in /sys/class/net/eth0/queues/rx-0/rps_cpus
Setting mask 00000003 in /sys/class/net/eth0/queues/rx-1/rps_cpus
Setting net.core.rps_sock_flow_entries to 32768
Setting limit 16384 in /sys/class/net/eth0/queues/rx-0/rps_flow_cnt
Setting limit 16384 in /sys/class/net/eth0/queues/rx-1/rps_flow_cnt
Trying to enable ntuple filtering HW offload for eth0...not supported
Setting mask 00000001 in /sys/class/net/eth0/queues/tx-0/xps_cpus
Setting mask 00000002 in /sys/class/net/eth0/queues/tx-1/xps_cpus
Writing '4096' to /proc/sys/net/core/somaxconn
Writing '4096' to /proc/sys/net/ipv4/tcp_max_syn_backlog
```

**Comment by vladzcloudius** [comment]
@slivne Please, let me know if we want to fix the 1.7? - it's a single line patch (redirect the output of `cd -` to /dev/null).

**Comment by slivne** [comment]
no need to backport 2.1 is almost out :)
